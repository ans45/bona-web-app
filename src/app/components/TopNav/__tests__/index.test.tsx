import React from 'react';
import { render, screen } from '@testing-library/react';

import { TopNav } from '..';

describe('<TopNav  />', () => {
  it('should match snapshot', () => {
    const topNav = render(<TopNav />);
    expect(topNav.container.firstChild).toMatchSnapshot();
  });

  it('should contain the site title', async () => {
    render(<TopNav />);
    const siteTitle = await screen.getByTestId('site-title');
    expect(siteTitle).toHaveTextContent('Bona');
  });
});
