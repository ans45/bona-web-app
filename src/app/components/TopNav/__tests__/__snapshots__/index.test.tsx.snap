// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TopNav  /> should match snapshot 1`] = `
<header
  class="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary MuiPaper-elevation4"
>
  <div
    class="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
  >
    <h1
      class="MuiTypography-root MuiTypography-h6"
      data-testid="site-title"
    >
      Bona
    </h1>
  </div>
</header>
`;
